### DISCUSSION TITLE
How to switch to new user name after renaming the crossout Loginname via Gaijin Website

### DISCUSSION THREAD

**Discussion by AndiiiHD** [moguevara/RFB_Tool_Suite#26] [Q&A]
Hi i just have renamed a while ago and don't know how to choose the new Nickname now. There is a (under settings) a menu where i can choose my new nickname, if i do this the app tells me to restart. But after the restart its still the old Username. What am i doin wrong? How can i change the Co_Driver to the new Nickname?

**Comment by moguevara** [comment]
Hey Man, 

Sorry you're having trouble, I thought I'd fixed this in v0.1.0.76 and we are on v0.1.0.79 now. 

The good news is CO_Driver should self correct after you play a while since it only looks at the last 10 session to find the user name. I'll take another look when I get a chance at the user name detection logic. Let me know if you keep having trouble and I'll see what I can do.

If you want to see how close you are to switching over you can check out "ValidUsers" in your config.json in your documents folder. I pick the user name that occurred most in the login log from game.log. 

![image](https://user-images.githubusercontent.com/15959605/189900731-8f335420-9e46-4c89-acc7-366308b97758.png)

Thanks,

Morgan

**Reply by AndiiiHD** [reply]
hi thank you for your Help i see the following for ValidUsers: 364 for the old Username and 40 for the new Username. What does this mean then? I could not swap the user. Its still the old Name. I am asking cause i would also like to know how it behaves with the CrossoutDB Upload cause i am using this function. Will the Data be merged - or will a new Profile appear for the new Username? Or will Data be upto date but only the "old Username" is shown in crossoutDB? Any would be ok for me, just interested how it is done. Thanks for your fabulous tool. I got a Patreon to support your Work :)

ah i see all names in the crossoutDB Profile - sorry i had missed that.

**Comment by AndiiiHD** [comment]
Hi Just wanted tonlet you know that the codriver App still only Starts with the old Account Name. If i Switch it in the Settings to use the new one and donthe App reboot it again comes Up with the old Name. But i want to thank you all for your Work i really Love the App. How can i use a custom Theme? I am patreon but don't know how to Change it. Sorry If this is a noob question ;)
